metrics/exporter.go

```diff
@@ -79,6 +79,29 @@ func UpdateExporterFromConfigMap(component string, logger *zap.SugaredLogger) fu
 	}
 }
 
+// UpdateExporterFromConfigMapWithDefaults returns a helper func that can be used to update the exporter
```

Comment:
Super nit but I find the name of this method a bit confusing. Are the Defaults here that then get overridden by the ConfigMap? I'd think Opts might be better, since you're providing the values to use to create the helper func?
UpdateExporterFromConfigMapWithOpts(opts ExporterOptions, logger...)
Also might wan to say that Component must not be specified but ConfigMap must not, which is also a bit confusing, since the ExporterOptions struct has this gem in it:
	// ConfigMap is the data from config map config-observability. Must be present.

Reply:
Make sense. I updated the comment to point out that ConfigMap must not be present because the value from the ConfigMap will be used instead.